swarm/fuse/swarmfs_test.go

```diff
@@ -209,7 +209,15 @@ type testAPI struct {
 	api *api.Api
 }
 
-func (ta *testAPI) mountListAndUnmount(t *testing.T) {
+func (ta *testAPI) mountListAndUnmountEncrypted(t *testing.T) {
+	ta.mountListAndUnmount(true, t)
```

Comment:
Nitpick, but AFAIK it is convention to keep `t` as a first parameter. I know we do it as last some places in our codebase, but it is weird.

Reply:
Ok, I just followed our code, but now i fixed (at least here): d4ed180eafb95bd04f7cce27b529e99a071ea346